plugins/vli/fu-vli-pd-device.c

```diff
@@ -22,6 +22,12 @@ struct _FuVliPdDevice {
  * Device has a PS186 attached via I²C.
  */
 #define FU_VLI_PD_DEVICE_FLAG_HAS_I2C_PS186 (1 << 0)
+/**
+ * FU_VLI_PD_DEVICE_FLAG_SKIPS_ROM:
```

Comment:
I think this needs documenting in plugins/vli/README.md too -- i.e. `Flags:skips-rom`

Reply:
While I was adding `Flags:skips-rom`, I went back and added in other flags. Please take a look when you have time. Thanks!